app/styles/components/_aptables.scss

```diff
@@ -0,0 +1,106 @@
+//
+// Aptible Table Component
+//
+.aptable {
+  border: 1px solid #E0E0E0;
+  box-shadow: 0px 2px 4px 0px rgba(0,0,0,0.06);
+  border-radius: 3px;
+  border-collapse: separate;
+  width: 100%;
+
```

Comment:
Thanks for bringing this up. After thinking about it... I may go for a new class that uses an `aptable` in its composition.

`aptable-group` perhaps? With an `aptable--no-border`. Going to think about it a bit more.